### PR TITLE
Don't display disclaimer prior to activation code use

### PR DESCRIPTION
**Pull request by Fang-**
Moves it to before the "download passport" step.
Still displays it prior to login.

### CODE REVIEW

File: src/Bridge.js [hunk 1]
```diff
@@ -48,13 +48,12 @@ function useInitialRoutes() {
...
-    : ROUTE_NAMES.LOGIN;
-
-  const extraRoutes = !hasDisclaimed ? [{ key: ROUTE_NAMES.DISCLAIMER }] : [];
-
-  return [{ key: landingPage }, ...extraRoutes];
+  return hasImpliedTicket
+    ? [{ key: ROUTE_NAMES.ACTIVATE }]
+    : [
+        { key: ROUTE_NAMES.LOGIN },
+        ...(!hasDisclaimed ? [{ key: ROUTE_NAMES.DISCLAIMER }] : []),
```
Comment (shrugs) [inline src/Bridge.js]: was this reverted or something? left version is much more readable

File: src/views/Activate/ActivateCode.js [hunk 2]
```diff
@@ -48,7 +50,10 @@ export default function ActivateCode() {
   const goToLogin = useCallback(() => history.popAndPush(ROUTE_NAMES.LOGIN), [
     history,
   ]);
-  const goToPassport = useCallback(() => push(names.PASSPORT), [names, push]);
+  const goToPassport = useCallback(() => {
+    push(names.PASSPORT);
+    if (!hasDisclaimed) push(names.DISCLAIMER);
```
Comment (shrugs) [inline src/views/Activate/ActivateCode.js]: nit: curly braces for simple branches are used basically everywhere else

File: src/Bridge.js (hunk 1 above)
Comment (Fang-) [reply src/Bridge.js]: This is doing something different though. It's only adding the disclaimer to the route for the login case, not the activate case. The previous version was adding it for both.

The js too fancy here?

Comment (shrugs) [reply src/Bridge.js]: ah, didn't notice the new logic. it's fancy, but I'd say it's a-ok

-matt

**Pull request merged**